app/javascript/packs/document-capture.jsx

```diff
@@ -86,6 +86,7 @@ loadPolyfills(['fetch', 'crypto']).then(async () => {
       <UploadContextProvider
         endpoint={appRoot.getAttribute('data-endpoint')}
         statusEndpoint={appRoot.getAttribute('data-status-endpoint')}
+        statusPollInterval={Number(appRoot.getAttribute('data-status-poll-interval'))}
```

Comment:
WDYT of adding some sort of obvious default? That way if we mess up configuration elsewhere we don't poll every 0ms?

```suggestion
        statusPollInterval={Number(appRoot.getAttribute('data-status-poll-interval')) || 1111}
```

My thinking is that we wouldn't normally configure something to 1111 so that if we did inspect and see that as a value, we'd know that it was this default

Reply:
> WDYT of adding some sort of obvious default? That way if we mess up configuration elsewhere we don't poll every 0ms?
> 
> My thinking is that we wouldn't normally configure something to 1111 so that if we did inspect and see that as a value, we'd know that it was this default

Good thinking. I was sort of relying on a loosely-held assumption that if this failed to parse (i.e. [`NaN`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN)), that our `setTimeout` wouldn't take it, but JavaScript being JavaScript, apparently `setTimeout(fn, NaN)` is treated as effectively the same as `setTimeout(fn, 0)` 🤦  Furthermore, I'd assumed that parsing any non-numeric value here would be `NaN` in the first place, but apparently `Number(null)` (where [`null` could easily be the result of a `getAttribute` for a missing attribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#Non-existing_attributes)) is not `NaN`, but instead `0` 🤦 

Since we already have the logic in place to [avoid scheduling a follow-up attempt if the interval is `undefined`](https://github.com/18F/identity-idp/blob/0808606fa4e7530b7106429f6dd6143dedbdd865/app/javascript/packages/document-capture/components/submission-complete.jsx#L35), and since I'd be more comfortable to fail loudly than to poll at all, maybe we should fall back to `undefined`.

```suggestion
        statusPollInterval={Number(appRoot.getAttribute('data-status-poll-interval')) || undefined}
```

Reply:
Yup the explicit `undefined` works for me! Yeah I tried out `Number()` in the console and was surprised that it actually....returned a number always 😂

Reply:
Falling back to `undefined` in 8cafb4a2f8759e377b25e032730b34609c105546 (mistakenly combined with other changes) with an extra "confidence-check" test case in d4ca919.